partcad/src/partcad/runtime_python_conda.py

```
@@ -48,77 +51,90 @@
-                                "pip",
-                            ],
-                            stdout=subprocess.PIPE,
-                            stderr=subprocess.PIPE,
-                        )
-                        _, stderr = p.communicate()
-                        if not stderr is None and stderr != b"":
-                            pc_logging.error(
-                                "conda pip install error: %s" % stderr
-                            )
+                    # Install new conda environment with the preferred Python version
+                    p = subprocess.Popen(
```

Ruff (0.8.2)

`subprocess` call: check for execution of untrusted input

(S603)